ext/vulnsrc/redhat/redhat.go

```diff
@@ -31,80 +31,93 @@ import (
 	"github.com/quay/clair/v3/database"
 	"github.com/quay/clair/v3/ext/versionfmt/modulerpm"
 	"github.com/quay/clair/v3/ext/versionfmt/rpm"
+	"github.com/quay/clair/v3/ext/vulnsrc"
 	"github.com/quay/clair/v3/pkg/commonerr"
+	"github.com/quay/clair/v3/pkg/envutil"
 	"github.com/quay/clair/v3/pkg/httputil"
-	"github.com/quay/clair/v3/ext/vulnsrc"
 	log "github.com/sirupsen/logrus"
 )
 
 const (
-	// PulpV2BaseURL - base url for pulp v2 content
-	PulpV2BaseURL            = "https://www.redhat.com/security/data/oval/v2/"
 	// PulpManifest - url suffix for pulp manifest file
-	PulpManifest             = "PULP_MANIFEST"
+	PulpManifest = "PULP_MANIFEST"
 	// DbManifestEntryKeyPrefix - key prefix used to create flag for manifest entry hash key/value
 	DbManifestEntryKeyPrefix = "oval.v2.pulp.manifest.entry."
 	// DbLastAdvisoryDateKey - key prefix used to create flag for last advisory date key/value
-	DbLastAdvisoryDateKey    = "oval.v2.advisory.date.issued"
+	DbLastAdvisoryDateKey = "oval.v2.advisory.date.issued"
 	// DefaultLastAdvisoryDate - literal date (in case no existing last advisory date is found)
-	DefaultLastAdvisoryDate  = "1970-01-01"
+	DefaultLastAdvisoryDate = "1970-01-01"
 	// AdvisoryDateFormat date format for advisory dates ('magical reference date' for datetime format)
-	AdvisoryDateFormat       = "2006-01-02"
+	AdvisoryDateFormat = "2006-01-02"
 	// UpdaterFlag - key used for flag for updater
-	UpdaterFlag              = "RedHatOvalV2Updater"
+	UpdaterFlag = "redHatUpdater"
 	// UpdaterFlagDateFormat - date format for updater flag dates ('magical reference date' for datetime format)
-	UpdaterFlagDateFormat    = "2006-01-02 15:04:05"
+	UpdaterFlagDateFormat = "2006-01-02 15:04:05"
 	// AffectedType - affected type
-	AffectedType             = database.BinaryPackage
+	AffectedType = database.BinaryPackage
 	// CveURL - url for cve content
-	CveURL                   = "https://access.redhat.com/security/cve/"
+	CveURL = "https://access.redhat.com/security/cve/"
 )
 
+// OvalV2BaseURL - base url for oval v2 content
+var OvalV2BaseURL = envutil.GetEnv("OVAL_V2_URL", "https://www.redhat.com/security/data/oval/v2/")
+
+// LogLevel - log level for oval v2 plugin
+var LogLevel = envutil.GetEnv("REDHAT_OVAL_V2_LOG_LEVEL", "info")
```

Comment:
Please don't change a log level inside a plugin. The log level is set for the whole app.

Reply:
removed plugin-specific log level treatment